### PR TITLE
Refactor CSV and JSONL importers and add error handling

### PR DESCRIPTION
**Pull request by berggren**
If uploaded file is malformed we report error back to the user and set the timeline and index as failed.

### CODE REVIEW

File: timesketch/lib/tasks.py [hunk 1]
```diff
@@ -96,76 +103,55 @@ def run_plaso(source_file_path, timeline_name, index_name, username=None):
...
-    total_events = es.import_event(index_name, event_type)
+    # Reason for the broad exception catch is that we want to capture
+    # all possible errors and exit the task.
+    try:
+        es.create_index(index_name=index_name, doc_type=event_type)
+        for event in read_and_validate(source_file_path):
+            es.import_event(index_name, event_type, event)
+        # Import the remaining events
+        total_events = es.import_event(index_name, event_type)
+    except Exception as e:
```
Comment (rbdebeer) [inline timesketch/lib/tasks.py]: todo (for future release): catch specific errors and handle.

**Comment by rbdebeer** [comment]
LGTM.

I see some jenkins build errors around pylint and timesketch not being able to be called, assume this is a config issue of some sort?

File: timesketch/lib/tasks.py (hunk 1 above)
Comment (berggren) [reply timesketch/lib/tasks.py]: Reason for the broad exception catch is that we want to capture
 -    for event in read_and_validate_csv(source_file_path):		 +    # all possible errors and exit the task.

**Comment by berggren** [comment]
Lint error in tasks.py (trailing newline). Fixed now.

**Pull request merged**